src/status_im/ui/screens/discover/views/search_results.cljs

```diff
@@ -49,7 +49,7 @@
        (if (empty? discoveries)
          [view st/empty-view
           ;; todo change icon
```

Comment:
we can remove this todo we have designers

Reply:
ok